src/ai/backend/manager/config.py

```diff
@@ -395,9 +397,9 @@
         t.Key(
             "threshold", default=_config_defaults["session"]["hang-tolerance"]["threshold"]
         ): t.Dict({
-            t.Key(SessionStatus.PREPARING.name, optional=True): tx.TimeDuration(),
-            t.Key(SessionStatus.TERMINATING.name, optional=True): tx.TimeDuration(),
-        }).ignore_extra("*"),
+            t.Key(status, optional=True): tx.TimeDuration()
+            for status in itertools.chain(SessionStatus, KernelStatus)
+        }).allow_extra("*"),
```

Comment:
> [!NOTE]
> This change allows all statuses, including `PENDING`, `RUNNING`, and others.

Reply:
I think session and kernel status should be tied to different keys.

Reply:
It seems like it would be difficult to manage the configuration of a dynamic key, is there any other way?

Reply:
I will leave the `SessionStatus.PREPARING` and `SessionStatus.TERMINATING` states as they are for now. Additionally, I will implement a kernel-collector to terminate kernels whose sessions have been terminated but the kernels themselves are still running. This will assist the session-collector cleanup context.